static/css/summary.less

```diff
@@ -30,8 +34,10 @@
           -----------------------
           .summary
```

Comment:
From other structural cheatsheets I've seen, the element is included, not just the class. E.g. https://github.com/cfpb/cfgov-refresh/blob/gh-pages/static/css/activity.less#L37